### DISCUSSION TITLE
Document small breaking change in 0.0.20 for sqlmodel.sql.sqltypes.GUID()

### DISCUSSION THREAD

**Discussion by fhuitelec** [fastapi/sqlmodel#1048] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Before

import sqlmodel.sql.sqltypes

column_type = sqlmodel.sql.sqltypes.GUID()

# After

import sqlalchemy.types

column_type = sqlalchemy.types.Uuid()
```

### Description

Hi!

I see that in [0.0.20](https://github.com/tiangolo/sqlmodel/releases/tag/0.0.20), you released UUID support through https://github.com/tiangolo/sqlmodel/pull/992 (and that's super cool! ✨).

I came upon an issue after upgrading to this version since the class `sqlmodel.sql.sqltypes.GUID` disappeared but it is used in existing alembic migrations.

I read https://github.com/tiangolo/sqlmodel/pull/992 in details and I get why it disappears and that you now return `sqlalchemy.types.Uuid` ([code](https://github.com/tiangolo/sqlmodel/pull/992/files#diff-d47aa94b4399636b3bf61c282806410ca16c47e6edb13a2d3ba675f9460e7389R611)).

However I'd suggest adding a **Breaking change** section to the release [0.0.20](https://github.com/tiangolo/sqlmodel/releases/tag/0.0.20) with a few details about how to migrate from `sqlalchemy.types.Uuid` or `sqlalchemy.types.UUID` to save time for others having the same issue upgrading.

Would you be open to this?

### Operating System

macOS

### SQLModel Version

0.0.16

### Python Version

3.11.8

### Additional Context

The issue I describe arises when migrating from 0.0.16 to 0.0.21 but I'm fairly sure this comes from [0.0.20](https://github.com/tiangolo/sqlmodel/releases/tag/0.0.20).

**Comment by setetiq** [comment]
Same here, alembic migrations are failing as GUID type is absent.

**Comment by pdjohntony** [comment]
Same. After upgrading sqlmodel I had to go back and modify my migration files from
```
sa.Column('id', sqlmodel.sql.sqltypes.GUID(), nullable=False),
```
to
```
sa.Column('id', sa.Uuid(), nullable=False),
```

**Reply by ogabrielluiz** [reply]
or 
```
sa.Column('id', sqlmodel.sql.sqltypes.types.Uuid(), nullable=False),
```
